website/docs/r/network_peering.html.markdown

```diff
@@ -111,11 +104,6 @@ resource "mongodbatlas_network_container" "test" {
   region           = "US_WEST"
 }
```

Comment:
It's missing one GCP resource to set the correct TF configuration. I placed examples with GCP and Azure on the PR #181, Maybe it would be great to update those examples, we can update them on the #181 PR what do you think?

Reply:
Ah, good point.   I could update the examples here from what you put in PR 181 (which helped me remember to do this update).    I'll do that now!

Reply:
Nice :) glad to help

Reply:
Added with related changes and tried to capture what you reported about GCP.  Please check it out @PacoDw.

Reply:
Thank you :)

Reply:
Travis marks two spelling errors in the following lines:
```bash
website/docs/r/network_peering.html.markdown:40:8: "RESOURSE" is a misspelling of "RESOURCES"
230website/docs/r/network_peering.html.markdown:149:41: "RESOURSE" is a misspelling of "RESOURCES"
```

Reply:
Thank you, fixed them and passed 💯